src/components/keypad/keypad-page-items.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import React from "react";
+import {View} from "@khanacademy/wonder-blocks-core";
+
+import Button from "./button";
+import ButtonAsset from "./button-assets";
+import type {KeyConfig} from "../../data/key-configs";
+
+import type {Node} from "React";
+
+const KeypadPageContainer = ({children}: {children: Node}) => (
+    <View
+        style={{
+            backgroundColor: "#DBDCDD",
+            width: "100%",
+            height: 192,
+
+            display: "grid",
+            gridTemplateColumns: "repeat(6, 1fr)",
+            gridTemplateRows: "repeat(4, 1fr)",
+        }}
```

Comment:
nit: extract into const styles?

Reply:
I don't have a super strong feeling but since it's one style for the whole component it felt ok to inline.

Reply:
Wow, what a unique color! I don't see it being used in webapp or wonder-blocks either.